docs/integrations/prefect-aws/ecs_guide.mdx

```diff
@@ -141,250 +180,579 @@ Save this policy to a file, such as `ecs-trust-policy.json`:
+                        "containerDefinitions": [
+                            {
+                                "name": "prefect-worker",
+                                "image": "prefecthq/prefect:3-latest",
+                                "cpu": 512,
+                                "memory": 1024,
+                                "essential": true,
+                                "command": [
+                                    "/bin/sh",
+                                    "-c",
+                                    "pip install prefect-aws && prefect worker start --pool my-ecs-pool --type ecs"
+                                ],
+                                "environment": [
+                                    {
+                                        "name": "PREFECT_API_URL",
+                                        "value": "prefect-api-url>"
+                                    },
+                                    { // [!code --:4]
+                                        "name": "PREFECT_API_KEY",
+                                        "value": "<prefect-api-key>"
+                                    },
+                                    { // [!code ++:4]
+                                        "name": "PREFECT_API_AUTH_STRING",
+                                        "value": "admin:yourpassword"
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                    ```
+                </Accordion>
+            </AccordionGroup>
+        </Accordion>                  
+        
+        Save the following JSON to a file named `task-definition.json`:
+        
+        ```json wrap
+        {
+            "family": "prefect-worker-task",
+            "networkMode": "awsvpc",
+            "requiresCompatibilities": [
+                "FARGATE"
+            ],
+            "cpu": "512",
+            "memory": "1024",
+            "executionRoleArn": "<ecs-task-execution-role-arn>",
+            "taskRoleArn": "<ecs-task-role-arn>",
+            "containerDefinitions": [
+                {
+                    "name": "prefect-worker",
+                    "image": "prefecthq/prefect:3-latest",
+                    "cpu": 512,
+                    "memory": 1024,
+                    "essential": true,
+                    "command": [
+                        "/bin/sh",
+                        "-c",
+                        "pip install prefect-aws && prefect worker start --pool my-ecs-pool --type ecs"
+                    ],
+                    "environment": [
+                        {
+                            "name": "PREFECT_API_URL",
+                            "value": "prefect-api-url>"
+                        },
+                        {
+                            "name": "PREFECT_API_KEY",
+                            "value": "<prefect-api-key>"
+                        }
```

Comment:
I think we should direct user to create an AWS secret for their ECS worker service and show how to expose the value as an environment variable.

Reply:
Agreed, there is already a callout that mentions using Secrets Manager for storing the API key.

Adding Secret Manager setup to this guide would add a lot of text to this guide which isn't relevant for Prefect-specific functionality. I'd like to earmark this for the "advanced" guide. WYT @desertaxle?

Reply:
I'm fine with linking out to docs for how to set up a secret, but I strongly believe that all our templates should show using a secret as the default option. Right now, it looks like we're recommending storing Prefect API keys in plaintext, which is a red flag.

If we updated this to something like
```suggestion
                        {
                            "name": "PREFECT_API_KEY",
                            "valueFrom": "<aws_secret_arn>"
                        }
```
I think that would work.